tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature

```diff
@@ -444,9 +444,8 @@ Feature: Share by public link
       | expireDate | +6          |
     And user "user1" has logged in using the webUI
     When the user edits the public link named "Public link" of file "lorem.txt" changing following
-      | expireDate | +8 |
-    Then the user should see an error message on the public link share dialog saying "Cannot set expiration date more than 7 days in the future"
-    And user "user1" should have a share with these details:
+      | expireDate | +15 |
+    Then user "user1" should have a share with these details:
```

Comment:
The user should actually see an error message here. Why was that `Then` phrase removed?

Reply:
The user cannot change the date in this case because phoenix got the correct maximum expire date already and so the user cannot save the change and no message comes.
@Talank please change this scenario using a step with `tries ...` that will make it clearer that we don't expect this to really change anything and please add a scenario where the admin changes the max date after login and with that the user is able to select an invalid date and will receive the error message